app/_hub/kong-inc/ai-proxy-advanced/overview/_index.md

```diff
@@ -92,7 +99,8 @@ The following upstream URL patterns are used:
 | Anthropic | `https://api.anthropic.com:443/{route_type_path}`                                                      |
 | Llama2    | As defined in `config.model.options.upstream_url`                                                      |
 | Mistral   | As defined in  `config.model.options.upstream_url`                                                     |
-
+| Amazon Bedrock   | Use the specific LLM upstream URL                                                   |
```

Comment:
Based on what we did for Gemini, I don't think this is right. We found the Gemini URL in a lua schema somewhere? But I don't remember where or which one and I think I saw the Bedrock one there as well, but couldn't remember what it was.